### PR TITLE
Addition of statechart and signals2 to boost_library

### PR DESCRIPTION
**Pull request by sanjeevcvh**
Addition of statechart and signals2 to boost_library

### CODE REVIEW

File: boost_libraries.bzl [hunk 1]
```diff
@@ -752,6 +752,59 @@ BOOST_LIBRARIES = [
     boost_library(
         name = "winapi",
     ),
+    
+    boost_library(
+        name = "conversion",
```
Comment (iceboy233) [inline boost_libraries.bzl]: Could you please sort the `boost_library` entries in alphabet order?

Comment (sanjeevcvh) [reply boost_libraries.bzl]: Agreed and will do the changes

File: boost_libraries.bzl [hunk 2]
```diff
@@ -625,6 +661,24 @@ BOOST_LIBRARIES = [
         deps = ["config"]
     ),
 
+    boost_library(
+        name = "statechart",
```
Comment (iceboy233) [inline boost_libraries.bzl:660]: nit: statechart < static_assert

**Review by iceboy233 — APPROVED** [review]
Note: I was thinking to add build test, to ensure build success, as new boost versions can change dependencies, which was seen in the past. Previously this was not a problem as all libraries are used by my repositories and I can use them to test boost upgrades. Let me know if you have any idea on how to implement such test. Thanks!

File: boost_libraries.bzl (hunk 2 above)
Comment (sanjeevcvh) [reply boost_libraries.bzl:660]: done the changes

**Comment by iceboy233** [comment]
Thanks!

**Pull request merged**